machinelearning/tensorflow.md

```diff
@@ -116,21 +117,17 @@ ck pull repo:ck-env
 
 # Download ImageNet's validation set
 # These will be installed to ${HOME}/CK_TOOLS/
-# Select option 1: val-min data set.
-ck install package --tags=image-classification,dataset,imagenet,aux
-ck install package --tags=image-classification,dataset,imagenet,val
+echo 0 | ck install package --tags=image-classification,dataset,imagenet,aux
+echo 1 | ck install package --tags=image-classification,dataset,imagenet,val
 
 # Copy the labels into the image location
-cp ${HOME}/CK-TOOLS/dataset-imagenet-ilsvrc2012-aux/val.txt ${HOME}/CK-TOOLS/dataset-imagenet-ilsvrc2012-val-min/val_map.txt
+cp ${HOME}/CK-TOOLS/dataset-imagenet-ilsvrc2012-aux-from.berkeley/val.txt ${HOME}/CK-TOOLS/dataset-imagenet-ilsvrc2012-val-min/val_map.txt
 
 cd inference/vision/classification_and_detection
 wget https://zenodo.org/record/2535873/files/resnet50_v1.pb
 
 # Install the additional packages required for resnet50 inference
-pip install opencv-python
-pip install pycocotools
-pip install psutil
-pip install tqdm
+pip install opencv-python pycocotools psutil tqdm
```

Comment:
Should you specify the minimum versions here for completeness?  i.e opencv-python>=X.Y?